frontend/src/lib/v2/WorkflowUtils.ts

```diff
@@ -19,6 +19,9 @@ import { convertFlowElements } from 'src/lib/v2/StaticFlow';
 import * as WorkflowUtils from 'src/lib/v2/WorkflowUtils';
 import { Workflow } from 'src/third_party/mlmd/argo_template';
 
+// This key can be used to retrieve 'pipeline_spec' in yaml file containing 'platform_spec'
```

Comment:
Why is this key being used to retrieved `'platform_spec'`? This key is for retrieving pipeline definition.

Reply:
I mean for the yaml file that contains 'platform_spec', we should use this key to retrieve `pipeline_spec`.